doc/sphinx/source/howto/tune-for-deployment.rst

```diff
@@ -0,0 +1,181 @@
+.. tune-for-deployment.rst
+
+
+############################################
+Fine-tune code deployment to nGraph devices  
+############################################
+
+When deploying code to nGraph-enabled devices, there are some additional 
```

Comment:
A problem with this section it is completely against what we are trying to do with nGraph -- we are failing very badly every time data scientists need to make their model different for different platforms. However, if someone is implementing a transformer this is important information.

Reply:
Mostly it's making the TFDO work the team did available across any framework that wants to use it.  This section is for framework developers who want to build systems and take the learnings of the bridge code work we did to their framework.